packages/block-editor/src/components/rich-text/index.js

```diff
@@ -376,14 +375,7 @@ class RichTextWrapper extends Component {
 			>
 				{ ( { isSelected, value, onChange, Editable } ) =>
 					<>
-						{ isSelected && multilineTag === 'li' && (
-							<ListEdit
-								onTagNameChange={ onTagNameChange }
-								tagName={ tagName }
-								value={ value }
-								onChange={ onChange }
-							/>
-						) }
+						{ isSelected && children && children( { value, onChange } ) }
```

Comment:
What else can it be, other than `undefined` or a function?

Reply:
I guess that's it. It looks like we only support render children at the moment so it would be nice to explicitly verify against function. I think we are doing it in other places but I don't think it's a pattern which has to be enforced. 🤷‍♂

Reply:
I'm not a big fan of type checking something expected. Not using a function is wrong, so it may error, rather than fail silently.

If you had (dumb example)

```js
function sum( a, b ) {
  return a + b;
}
```

You also wouldn't type check the arguments?

Reply:
It's a super minor thing, it's fine as is 👍 Many would argue that TypeScript is the way to go in both cases 😃